contrib/intel/jenkins/runtests.py

```diff
@@ -0,0 +1,64 @@
+import argparse
+import os
+import sys
+sys.path.append(os.environ['CI_SITE_CONFIG'])
+import ci_site_config
+import run
+import common
+
+parser = argparse.ArgumentParser()
+rser.add_argument("--prov", help="core provider", choices=["psm2", "verbs", \
+                     "tcp", "udp", "sockets", "shm"])
+parser.add_argument("--build_mode", help="specify the build configuration", \
+                     choices = ["dbg", "dl"])
+parser.add_argument("--fi_info_test", help="run the fi_info test", \
+                    choices=['yes','no'])
```

Comment:
because there is no option for testname, as all tests except this one are run with the provider option specified and fi_info does not take provider argument.

Reply:
Hmm. fi_info should take a provider argument.